bin/o2a

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Entry script for the o2a main function"""
+from os import path
+
+import sys
+
+sys.path.insert(0, path.abspath(path.join(path.dirname(__file__), path.pardir)))
+
+if sys.version_info.major < 3 or (sys.version_info.major == 3 and sys.version_info.minor < 6):
```

Comment:
Może tego check-a powinniśmy mieć jednak w samym `o2a.o2a`? Bo jeśli ktoś odpali tamten skrypt bezpośrednio to się sprawdzenie nie uruchomi.

Reply:
It won't work in there - because it has some python3-specific code, so it will fail with import error :(.

Reply:
Gotcha, let's leave it as is then.